job-descriptions/software-engineer.md

```diff
@@ -4,10 +4,12 @@
 
 ### About the company
 
-From sustainable energy systems to self-driving cars, the innovations of the future will all rely on code. With so much software to write, developers need a better way to access, understand, and collaborate on code.
+At Sourcegraph, we are building a better, smarter foundation for software development. The innovations of the future will increasingly rely on code, so if we can make software developers more productive, we can literally bring the future sooner.

```

Comment:
What are your thoughts on:
>The innovations of the future will all rely on code. By empowering software developers today, we can literally bring the future sooner.

?

Reply:
I feel like those two sentences should be joined together with `and` or `so`.

> The innovations of the future will all rely on code and by empowering software developers today, we can literally bring the future sooner.

Thoughts?

Reply:
Sorry for the delay, started typing this up last week but got cut off:
===
Eh, they're two different clauses in my mind. But I'm fine putting it in one sentence. I'd prefer `so` to `and`, though — to me, it's a logic statement:

`A`: "At Sourcegraph, we are building a better, smarter foundation for software development." 
`B`: "The innovations of the future will all rely on [software development]."
`C`: "We can literally bring the future sooner"

`(A & B) -> C`

I feel that `so` captures "implies" better than `and` does.